float64 allowable_cost_per_observation
---
float64 final_cost_per_observation
geometry_msgs/Pose target_pose